developer-docs-site/docs/tutorials/testnet-devnet-incentivized-testnet.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Testnet, Devnet and Incentivized Testnet"
+slug: "testnet-devnet-and-incentivized-testnet"
+---
+
+# Testnet, Devnet and Incentivized Testnet
+
+You can connect to Aptos Blockchain in a few ways. See the below table:
+
+:::tip
+
+Make sure to see the row describing **What not to do**.
+
+:::
+
+|Description |Aptos Devnet | Testnet | Aptos Incentivized Testnet (AIT)|
+|---|---|---|---|
+|**Network runs where**| Runs on Aptos servers.| Runs locally on your computer, entirely independent from devnet. | Runs on Aptos servers.|
+|**Who is responsible for the network**|Managed by Aptos Team. | Deployed, managed and upgraded by you.| Managed by Aptos Team.|
+|**Purpose of the network**|The devnet is built for experimenting with new ideas, improve performance and enhance the user experience.| The testnet is for your development purposes and runs on your local computer.| For executing the Aptos Incentivized Testnet programs for the community.|
+|**Network status**|Mostly live, with brief interruptions during regular updates. | On your local computer. | **Live only during Incentivized Testnet drives**. |
+|**Type of nodes**|Validator nodes on Aptos servers. | Validator node on your local computer. | AIT FullNode and AIT validator nodes on your local computers.|
+|**How to connect to the network**|<ul><li> You can transact directly with devnet without a local testnet. See, for example, [Your first transaction](/tutorials/first-transaction.md).</li><li> You can run FullNode locally on your computer and synchronize with devnet. See [Run a FullNode](/tutorials/full-node/run-a-fullnode.md).</li></ul>| You can start a validator node locally and connect with your local testnet for development purposes. | You must start both a local AIT FullNode and a local AIT validator node to connect to the AIT-1.|
```

Comment:
nits:
- Devnet: "You can run FullNode locally" -> "You can run FullNodes locally"
- Local Testnet: "You can start a validator node locally..." -> "You can start a validator and Fullnode locally..."
- AIT: "You must start both a local AIT FullNode and a local AIT validator node to connect to the AIT-1." -> "You must start a local AIT validator node to connect to the AIT-1. FullNodes are optional and can also be connected to AIT-1"